app.js

```diff
@@ -18,17 +18,18 @@ if (process.env.NODE_ENV === 'test') {
   logger.child = () => {return {info: noop, error: noop, debug: noop};};
 }
 
-srf.connect(config.get('drachtio'));
-srf.on('connect', (err, hp) => {
-  if (err) throw err;
-  logger.info(`connected to drachtio listening on ${hp}`);
-});
-if (process.env.NODE_ENV !== 'test') {
```

Comment:
I still think we still want these two lines which log connection errors, except during tests